### DISCUSSION TITLE
i love charting songs i make

### DISCUSSION THREAD

**Discussion by stripes10122** [JordanSantiagoYT/FNF-JS-Engine#698] [General]
screw meek engine, js enguine is way better, more spam, meek engine doesnt even haev fat notes bruh, onlky zooms up to 8, js engine is better, and also its ez, im going to add more stuff

**Comment by moxie-coder** [comment]
are you even old enough to be on GitHub? How old are you

**Reply by stripes10122** [reply]
my keyboard sometimes doesnt enter letters, and i type fast

**Reply by moxie-coder** [reply]
okay
